src/Microsoft.DocAsCode.Metadata.ManagedReference.Roslyn/Visitors/CSYamlModelGenerator.cs

```diff
@@ -1077,6 +1077,14 @@ public static ExpressionSyntax GetLiteralExpressionCore(object value, ITypeSymbo
                     return SyntaxFactory.LiteralExpression(
                         SyntaxKind.NumericLiteralExpression,
                         SyntaxFactory.Literal((ulong)value));
+                case SpecialType.System_IntPtr:
```

Comment:
`nint x = 5;` is valid in C# 9. `IntPtr x = 5` is not so there may be some other tweak needed to ensure it is rendered correctly, but I wasn't able to determine where.